src/Mapbender/CoreBundle/Resources/public/mapbender.element.redlining.js

```diff
@@ -37,6 +37,15 @@
             }
             this._trigger('ready');
             this._ready();
+
+            //put rl layer on top of layer stack if a source is added, i.e. by wms loader
```

Comment:
I would prefer to setup such event listeners in a separate function. If it needs a comment for the following lines - please put these lines in a separate function also. This way the reading of the code and the widget inheritance will be much easier.